bpf/lib/common.h

```diff
@@ -447,6 +447,16 @@ struct node_value {
 	__u8  pad;
 };
 
+struct device_key {
+	__u32           ifindex;
+};
+
+struct device_value {
+	union macaddr   mac;
+	__u8            l3;
+	__u8		pad[7];
```

Comment:
Did you mean:

```suggestion
	__u8		l3:1;
	__u8		pad:7;
```

Reply:
pad[7] is for C and GO structs alignment. Without it, alignment check will fail.
```
C and Go structs alignment check in datapath failed: type DeviceValue's binary.Size (9) does not equal its unsafe.Sizeof (16) size
```